documentation/howto/os3_deployimage.adoc

```diff
@@ -0,0 +1,90 @@
+= Deploy a Docker Image to OpenShift 3
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+:toc:
+
+JBoss Tools' OpenShift 3 tooling allows users to deploy a Docker image to an OpenShift cluster from a Docker daemon.
+Deploying an image to an OpenShift cluster generates several resources to support the image which include:
+
+- ImageStream
+- Deployment Configuration
+- Service
+- Route 'Optional'
+
+[Note] 
+==================================================
+The following steps require:
+
+- Connection to a Docker daemon already exist
+- Connection to an OpenShift 3 cluster already exist
+- Openshift 3 project exist
```

Comment:
I can add that since I don't know how a service will get validated by the server when it gets uploaded, but you can create a pod for an image that doesnt expose ports.  Also, if the user doesnt explicitly add an 'EXPOSE' directive to their DockerFile, we won't know about exposed ports. This is very limiting for now because we have not added the pending feature to allow you to arbitrarily add a port in this flow.  I'll add a blurb about the ports.